web/styles/rendered_markdown.css

```diff
@@ -272,6 +272,13 @@
                    spoiler messages to wrap. */
                 flex: 1 1 auto;
             }
+
+            time {
+                /* Time tag pushes the arrow out of view when overflow,
```

Comment:
This looks acceptable, but be sure to clean up your comment here so it reads as proper English.

Reply:
sure will keep in mind thanks